### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by iangilman**
Fixes: https://github.com/openseadragon/openseadragon/issues/2245

Note that we're disabling the navigator tests for now; I'll file an issue to follow up on them.

### CODE REVIEW

File: test/modules/viewport.js [hunk 1]
```diff
@@ -531,14 +533,14 @@
...
             viewer.removeHandler('open', openHandler);
             var viewport = viewer.viewport;
-            viewport.setRotation(45, true);
+            viewport.setRotation(45, null, true);
             viewport.fitBounds(new OpenSeadragon.Rect(1, 1, 1, 1), true);
             viewport.applyConstraints(true);
             var bounds = viewport.getBounds();
             Util.assertRectangleEquals(
                 assert,
+                new OpenSeadragon.Rect(1.2071067811865466, 0.20710678118654746, Math.sqrt(2), Math.sqrt(2), 45),
```
Comment (iangilman) [inline test/modules/viewport.js:542]: @pearcetm With https://github.com/openseadragon/openseadragon/pull/2249, I suspect you may need to update these numbers. Please take a look!

File: test/modules/viewport.js [hunk 2]
```diff
@@ -555,15 +557,15 @@
...
             viewport.setFlip(true);
-            viewport.setRotation(45, true);
+            viewport.setRotation(45, null, true);
 
             viewport.fitBounds(new OpenSeadragon.Rect(1, 1, 1, 1), true);
             viewport.applyConstraints(true);
             var bounds = viewport.getBounds();
             Util.assertRectangleEquals(
                 assert,
+                new OpenSeadragon.Rect(1.2071067811865466, 0.20710678118654746, Math.sqrt(2), Math.sqrt(2), 45),
```
Comment (iangilman) [inline test/modules/viewport.js:567]: @pearcetm Likewise here.

File: test/modules/viewport.js (hunk 1 above)
Comment (pearcetm) [reply test/modules/viewport.js:542]: See https://github.com/openseadragon/openseadragon/pull/2249 for fix

File: test/modules/viewport.js (hunk 2 above)
Comment (pearcetm) [reply test/modules/viewport.js:567]: See https://github.com/openseadragon/openseadragon/pull/2249 for fix

**Pull request merged**